babel/src/main/java/org/apache/calcite/sql/babel/SqlTableAttributeIsolatedLoading.java

```diff
@@ -20,24 +20,24 @@
 import org.apache.calcite.sql.parser.SqlParserPos;
 
 /**
- * A <code>SqlCreateAttributeIsolatedLoading</code> is a CREATE TABLE option
+ * A <code>SqlTableAttributeIsolatedLoading</code> is a table option
  * for the WITH ISOLATED LOADING attribute.
  */
-public class SqlCreateAttributeIsolatedLoading extends SqlCreateAttribute {
+public class SqlTableAttributeIsolatedLoading extends SqlTableAttribute {
 
   private final boolean nonLoadIsolated;
   private final boolean concurrent;
   private final OperationLevel operationLevel;
 
   /**
-   * Creates a {@code SqlCreateAttributeIsolatedLoading}.
+   * Creates a {@code SqlTableAttributeIsolatedLoading}.
    *
    * @param nonLoadIsolated  Defines table as non-load isolated
```

Comment:
Java doc type consistency wise, would you make your new java classes to have the same indentation or is it not quite relevent.

Reply:
Can you link to an example that needs to be changed?

Reply:
Do you mean the param indentation? This class actually already existed, I just changed its name. I tried to keep param indentation justified for the classes I added, but I'm not sure it's worth going back and doing this for all the classes that were already written.

Reply:
From the other comment above, I think Arthur is referring to the fact that not all param's description sentences are aligned in this class whereas they are aligned in, say, SqlAlterTableAttributeFreeSpace.

I agree with @dstekol that since he is just porting existing classes, it's probably not necessary to go back and make sure all of the existing javadocs match the style of the new classes (e.g. SqlAlterTableAttributeFreeSpace). 

Furthermore, we have not been extremely pedantic about javadoc styling (e.g. number of spaces after param names, etc). But I think this is ok.